src/Sylius/Bundle/OrderBundle/Controller/OrderController.php

```diff
@@ -133,9 +133,11 @@ public function saveAction(Request $request): Response
 
     private function resetChangesOnCart(OrderInterface $cart): void
     {
-        $this->manager->refresh($cart);
-        foreach ($cart->getItems() as $item) {
-            $this->manager->refresh($item);
+        if ($this->manager->contains($cart)) {
+            $this->manager->refresh($cart);
+            foreach($cart->getItems() as $item) {
+                $this->manager->refresh($item);
+            }
         }
```

Comment:
```suggestion
        if (!$this->manager->contains($cart)) {
        	return;
        }
        
        $this->manager->refresh($cart);
        foreach($cart->getItems() as $item) {
            $this->manager->refresh($item);
        }
```

Return Early Pattern is usually more readable.